### DISCUSSION TITLE
Unable to upload files larger than 26 MB

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#4696] [Self Hosting] [answered]
Unable to upload any files above 26 MB, the limit should be increased and manually set within the settings menu.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to any document
2. Drag and drop any file above 26 MB to the document.
4. See error message

**Expected behaviour**
The file uploads successfully.

**Screenshots**
![image](https://user-images.githubusercontent.com/69816494/212348102-1a41a484-4af1-4051-8bfe-a1688464edb6.png)

**Outline (please complete the following information):**
- Install: self hosted

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.10
 - Browser: Google Chrome
 - Version: 109.0.5414.74

**Comment by tommoor** [comment, marked as answer]
`AWS_S3_UPLOAD_MAX_SIZE` env config.
